sentry-quartz/src/main/java/io/sentry/quartz/SentryJobListener.java

```diff
@@ -52,14 +52,15 @@ public void jobToBeExecuted(final @NotNull JobExecutionContext context) {
       if (maybeSlug == null) {
         return;
       }
-      final @NotNull ISentryLifecycleToken lifecycleToken = scopes.pushIsolationScope();
+      final @NotNull ISentryLifecycleToken lifecycleToken =
```

Comment:
We're not preferring `.forkedScopes().makeCurrent()` and `.forkedCurrentScope().makeCurrent()` over `pushIsolationScope()` and `pushScope()` as the fork methods allow us to specify a name/reason for forking, making debugging easier. Our own integrations should do it this way, whereas customers can simply use the push methods. Maybe we should also consider marking the fork methods internal.